controllers/platformoperator_controller.go

```diff
@@ -37,12 +40,14 @@ import (
 	"github.com/openshift/platform-operators/internal/util"
 )
 
+var (
+	ErrSourceFailed = errors.New("failed to run sourcing logic")
```

Comment:
Nit: Unexport if not meant to be consumed outside this package.
```suggestion
	errSourceFailed = errors.New("failed to run sourcing logic")
```

Reply:
Updated in https://github.com/openshift/platform-operators/pull/41/commits/28b4316bc8c76cdb4d61f41877fbe6351ac50f39.